include/mbedtls/ssl.h

```diff
@@ -84,8 +84,13 @@
 #define MBEDTLS_ERR_SSL_BAD_CERTIFICATE                   -0x7A00
 /* Error space gap */
 /**
```

Comment:
Testing: with the merge of https://github.com/Mbed-TLS/mbedtls/commit/6179f6d456ba94384292cd282cf3359bc3ba1d12 and https://github.com/Mbed-TLS/mbedtls/pull/9505/commits/f0eaaa01c12807fce59832fcc5a1a27bf58aa31b, `programs/ssl/tls13_client_demo.sh` passes.